sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -35,6 +25,17 @@ internal static TextAnalyticsError ConvertToError(Error error)
                 : new TextAnalyticsError(error.Code.ToString(), error.Message, error.Target);
         }
 
+        internal static List<TextAnalyticsError> ConvertToErrors(IReadOnlyList<Error> internalErrors)
```

Comment:
who is using this function?

Reply:
It is currently not being used - I wrote it because we used to have one for TextAnalyticsErrorInternal on line 21. Maybe it is needed in the LROs?

Reply:
Hmm maybe. ok later on we can delete it if we realize no one used it